excellent/legacy.go

```diff
@@ -155,8 +202,7 @@ var functionTemplates = map[string]functionTemplate{
 	"time": {name: "time", params: "(%s %s %s)"},
 }
 
-func newRootVarMapper() *varMapper {
-
+func newMigrationBaseVars() map[string]interface{} {
```

Comment:
all so that the root var mapper can be parameterized (i.e. the different ways to map `@extra`), without having to re-initialize this giant map everytime